modules/standard/GMP.chpl

```diff
@@ -304,6 +437,10 @@ module GMP {
   }
 
   /*
+    The BigInt class provides a more Chapel-friendly interface to the
+    GMP integer functions. In particular, this class supports GMP
+    numbers that are stored in distributed arrays.
+    
     All methods on BigInt work with Chapel types. Many of them use the gmp
     functions directly, wich use C types. Runtime checks are used to ensure the
```

Comment:
I would make the third paragraph a little more explicit, by saying "controlled by the flags" and maybe treat each flag the way you have in #1655 (I think).  I would also make the TODO section sound a little more aimed at users, or move it into a separate comment.